docker/start_container.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+echo "Starting container..."
+if [ -z $(docker ps --filter=name='citrixitm_tf_dev_container' -q) ]
+then
+    if [ -z $(docker ps --filter=name='citrixitm_tf_dev_container' -a -q) ]
+    then
+        echo "The citrixitm_tf_dev_container container doesn't exist. Did you mean \"make docker-run\"?"
+        RC=1
```

Comment:
`RC` is set but no used

Reply:
Ah, those should have been `exit 1`. Fixed.